src/specs/specs_parser.tsx

```diff
@@ -1,26 +1,37 @@
-import { inject } from 'mobx-react';
-import { PureComponent } from 'react';
-import { ChartStore } from '../chart_types/xy_chart/store/chart_state';
+import React, { useEffect } from 'react';
+import { bindActionCreators, Dispatch } from 'redux';
+import { connect } from 'react-redux';
+import { specParsed, specUnmounted } from '../state/actions/specs';
 
-export interface SpecProps {
-  chartStore?: ChartStore; // FIX
-}
+export const SpecsParserComponent: React.FunctionComponent<{}> = (props) => {
```

Comment:
I think the type here is not necessary 

```suggestion
export const SpecsParserComponent = (props: DispatchProps) => {
```

Reply:
It's required to type the `children` property of props